3.12-pytorch/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM articulate/python:3.12
```

Comment:
All images build at the same time, so we won't get the latest updates if we build off this

Reply:
Just updated it to not reference the existing Python image :+1: